### PR TITLE
[uss_qualifier] Optimize RID tests

### PR DESCRIPTION
**Pull request by BenjaminPelletier**
Currently, RID tests take a long time and we perform two of them in series (F3411-22a + U-space Remote Identification).  This PR makes an effort to reduce the time these tests take by shortening the simulated flights and bringing in the lead time between injection and monitoring.  It also removes the separate F3411-19 CI job in favor of resolving the TODO to add the netrid_v19 test to the standard set of tests.

In the process of making these changes, a number of issues were resolved or improved:

* In the sequence view artifact, unattributed (no associated participant IDs) events are now handled more clearly
* The time each scenario took is included on the sequence view artifact overview page
* The duration of the injected NetRID flights is noted in the report upon injection
* A bug in display_data_evaluator is fixed
* Additional information is printed when a duplicate telemetry error is encountered in display_data_evaluator
* The scenario query log message now includes the time the query took (to potentially visually identify long queries)

### CODE REVIEW

File: monitoring/uss_qualifier/reports/templates/sequence_view/overview.html [hunk 1]
```diff
@@ -156,8 +156,15 @@ <h2>Scenarios executed</h2>
       {% endif %}
       <th>Scenario</th>
       {% for participant_id in all_participants %}
-        <th>{{ participant_id }}</th>
+        <th>
+          {% if participant_id != "unattributed" %}
```
Comment (mickmis) [inline monitoring/uss_qualifier/reports/templates/sequence_view/overview.html:160]: I'm not familiar with this templating, but is it possible to refer to the variable from here?

**Review by mickmis — APPROVED** [review]
LGTM - nice improvements

File: monitoring/uss_qualifier/reports/sequence_view.py [hunk 2]
```diff
@@ -376,12 +398,23 @@ def append_notes(new_notes):
         else:
             post_notes = {}
         if post_notes:
+            latest_step_time = max(v.timestamp.datetime for v in post_notes.values())
             append_notes(post_notes)
 
+    if "end_time" in report and report.end_time:
+        latest_step_time = report.end_time.datetime
```
[the remark this answers is not in the document]
Comment (BenjaminPelletier) [reply monitoring/uss_qualifier/reports/sequence_view.py:405]: Yes, currently end_time is populated only [upon nominal completion of the scenario](https://github.com/interuss/monitoring/blob/2110d532485c0625aed07f672961ffb4d54d48e5/monitoring/uss_qualifier/scenarios/scenario.py#L410) which doesn't happen if scenario execution is interrupted (e.g., due to a High or Critical severity failed check), which is why [the end_time is optional](https://github.com/interuss/monitoring/blob/2110d532485c0625aed07f672961ffb4d54d48e5/monitoring/uss_qualifier/reports/report.py#L232) and therefore cannot be assumed to be populated.  I'm not sure this is/was the best choice, but it's how things work now so to approximate end_time we have to look at the latest thing that actually happened.

I would definitely be open to a PR that guaranteed population of end_time and made it mandatory, but that may not be the highest-value improvement.

File: monitoring/uss_qualifier/reports/templates/sequence_view/overview.html (hunk 1 above)
Comment (BenjaminPelletier) [reply monitoring/uss_qualifier/reports/templates/sequence_view/overview.html:160]: Yes; the variable has to be passed into the templating engine specifically, but that's probably still a good idea.  Will do.

File: monitoring/uss_qualifier/reports/sequence_view.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (mickmis) [reply monitoring/uss_qualifier/reports/sequence_view.py:405]: Oh I see - thanks for the explanation.

**Pull request merged**